Apps/WebClient/src/NewClientApp/src/components/site/NotificationCentreComponent.vue

```diff
@@ -0,0 +1,227 @@
+            <v-row align="center" class="align-self-start flex-grow-0 w-100">
+                <v-col cols="auto" class="text-primary">
+                    <HgIconButtonComponent
+                        data-testid="notification-centre-close-button"
+                        icon="angle-double-right"
+                        size="x-small"
+                        @click="
+                            notificationStore.isNotificationCenterOpen = false
+                        "
+                    />
+                </v-col>
+                <v-col>
+                    <h5 class="text-body-1 text-primary">
+                        Notification Centre
+                    </h5>
+                </v-col>
+                <v-col v-if="notifications.length > 0" cols="auto">
+                    <HgButtonComponent
+                        data-testid="notification-centre-dismiss-all-button"
+                        variant="link"
+                        color="grey"
+                        text="Clear All"
+                        :loading="isMessageLoading()"
+                        @click="dismissAllNotifications"
+                    />
+                </v-col>
+            </v-row>
+            <v-row
+                v-if="notifications.length === 0"
+                align="center"
+                justify="center"
+            >
+                <v-col>
+                    <v-img
+                        src="@/assets/images/home/empty-state-notifications.svg"
+                        alt="No Notifications"
+                    />
+                    <p class="text-center mt-4 text-body-1">No Notifications</p>
+                </v-col>
+            </v-row>
+            <v-row
+                v-else
+                class="flex-column w-100"
+                data-testid="notifications-div"
+            >
+                <v-col
+                    v-for="notification in notifications"
+                    :key="notification.id"
+                    class="flex-grow-0 py-2 px-0"
+                >
+                    <p
+                        class="text-body-2"
+                        :class="{ 'text-grey-lighten-1': !isNew(notification) }"
+                    >
+                        {{ formatDate(notification.scheduledDateTimeUtc) }}
+                    </p>
+                    <v-card class="bg-grey-lighten-5">
+                        <v-card-title class="text-right pa-0">
+                            <HgIconButtonComponent
+                                :data-testid="`notification-${notification.id}-dismiss-button`"
+                                class="text-muted"
+                                icon="xmark"
+                                size="small"
+                                :loading="isMessageLoading(notification.id)"
+                                @click="dismissNotification(notification.id)"
+                            />
+                        </v-card-title>
```

Comment:
Positioning the close button in its own row leaves a pretty noticeable gap above the start of the text.  This could be balanced visually by adding a corresponding gap below the text, and perhaps on the sides too.